akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -793,8 +795,21 @@ private[remote] class EndpointManager(conf: Config, log: LoggingAdapter) extends
       "endpointWriter-" + AddressUrlEncoder(remoteAddress) + "-" + endpointId.next()))
   }
 
+  private var normalShutdown = false
```

Comment:
so many things are in this class, I wanted to keep it close to the most important use-site.